src/modals/import-account-modal/import-account-seed/import-account-seed.tsx

```diff
@@ -45,7 +46,9 @@ export const ImportAccountSeed: FC<Props> = ({ onBackHandler }) => {
     const seed = mnemonicToSeedSync(values.seedPhrase);
     const privateKey = seedToHDPrivateKey(
       seed,
-      values.derivationType === ImportAccountDerivationEnum.DEFAULT ? getDerivationPath(0) : values.derivationPath
+      values.derivationType === ImportAccountDerivationEnum.DEFAULT || !isString(values.derivationPath)
+        ? getDerivationPath(0)
+        : values.derivationPath
```

Comment:
does it's correct that you use `getDerivationPath(0)` when `!isString(values.derivationPath)`?  
As I remember it's different logic on extension.